themes/src/main/resources/theme/keycloak.v2/account/src/app/content/signingin-page/SigningInPage.tsx

```diff
@@ -383,29 +496,44 @@ class CredentialAction extends React.Component<CredentialActionProps> {
     render(): React.ReactNode {
         if (this.props.updateAction) {
             return (
-                <DataListAction aria-labelledby='foo' aria-label='foo action' id={'updateAction-' + this.props.credential.id}>
-                    <Button id={`${SigningInPage.credElementId(this.props.credential.type, this.props.credential.id, 'update')}`} variant='primary'onClick={()=> this.props.updateAction.execute()}><Msg msgKey='update'/></Button>
+                <DataListAction
+                    aria-labelledby="foo"
+                    aria-label="foo action"
```

Comment:
It's not allowed to have both an `aria-labelledby` and `aria-label` attribute, they are mutually exclusive. Also an `aria-label` attribute has to be a human readable text, as it's what the screen reader will read to the user.

Reply:
So oddly both are required on the DataListAction in the DataList component from PF. In which case, yeah the `labeledby` will take precedence. I've updated all these to be translatable, screen reader readable, strings.